cmd_tools/create_hdusd_libs.py

```diff
@@ -82,11 +82,11 @@ def main():
 
     # Add the arguments to the parser
     ap.add_argument("-usd", required=True, metavar="",
-                    help="USD dir")
+                    help="USD build dir")
```

Comment:
are we building against the source build dir or the "built" usd dir?

Reply:
against "built" usd dir